scripts/check-release-notes.sh

```diff
@@ -6,6 +6,13 @@ set -e
 # format of the string stored in the variable is: refs/pull/:PRNUMBER/merge.
 PR_NUMBER=$(echo $GITHUB_REF | awk 'BEGIN { FS = "/" } ; { print $3 }')
 
+# If this is a PR being merged into the main repo, then the PR number will
+# actually be "master" here, so we'll ignore this case, and assume that in
+# order for it to be merged it had to pass the check in its base branch.
+if [$PR_NUMEBR == "master"]; then
```

Comment:
Bash is very strict here. You need spaces surrounding `[` and `]`, otherwise you get an error like `line 12: [: missing `]'`.

Reply:
```suggestion
if [$PR_NUMBER == "master"]; then
```